tests/python/contrib/test_ethosu/cascader/test_graph.py

```diff
@@ -16,14 +16,9 @@
 # under the License.
 import pytest
 
-from tvm.contrib.ethosu.cascader import (
-    StripeConfig,
-    Propagator,
-    Tensor,
-    InlinePart,
-    TESubgraph,
-    CascaderGraph,
-)
+pytest.importorskip("ethosu.vela")
+
+import tvm.contrib.ethosu.cascader as cs
```

Comment:
As much as I appreciate using my initials here, it'd be really great to use something I could immediately associate with the cascader; potentially `cascader` rather than `cs` ?